Dockerfile-tests

```diff
@@ -0,0 +1,14 @@
+FROM bigtruedata/sbt
```

Comment:
Do we need a `Dockerfile-tests` if it's all in the same sbt project? :)

Reply:
I think we do unless we revamp how we run automation tests in ci, which I think might not be a bad idea, but I think that's another ticket